client/services/code_evaluators/PythonCodeRunnerService.js

```diff
@@ -38,12 +39,17 @@ tie.factory('PythonCodeRunnerService', [
         return Sk.misceval.asyncToPromise(function() {
           return Sk.importMainWithBody('<stdin>', false, code, true);
         }).then(function() {
+          var resultList = [];
+          if (Sk.globals.hasOwnProperty(VARNAME_TEST_RESULTS)) {
+            resultList = Sk.ffi.remapToJs(Sk.globals[VARNAME_TEST_RESULTS]);
```

Comment:
What does this do? Might be worth commenting for posterity, especially for those of us that are less JavaScripty. :)